.github/workflows/tests.yml

```diff
@@ -0,0 +1,66 @@
+# TODO: Line 20, enable python-version: 3.5
+# TODO: Line 69, enable pytest --doctest-modules
+
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [10.x, 12.x, 14.x]
+        python: [3.6, 3.7, 3.8]
```

Comment:
Please do just one test run on `3.9.0-rc.1` on your own to make sure that we are ready for the Py39 release in one month.

Reply:
tested it in cc9942e and it passes (only the doctest-modules fails so I reverted it back to a TODO)